climada/util/coordinates.py

```diff
@@ -944,6 +944,35 @@ def country_iso_alpha2numeric(isos):
         nums.append(num)
     return nums[0] if return_int else nums
 
+def country_iso_numeric2alpha(isos_numeric):
+    """Convert ISO 3166-1 numeric-3codes to alpha-3
+    
+    Parameters
+    ----------
+    isos_numeric : int or list of int
+        ISO numeric-3 codes of countries (or single code).
+        
+    Returns
+    -------
+    isos : str or list of str
+        Will only return a list if the input is a list.
+    """
+    return_str = isinstance(isos_numeric, int)
+    isos_numeric = [isos_numeric] if return_str else isos_numeric
+    old_iso = {
+        0: '',  # Ocean or fill_value
+        530: "ANT", # Netherlands Antilles: split up since 2010
+        891: "SCG", # Serbia and Montenegro: split up since 2006
+    }
```

Comment:
There are several more, such as the URSS, Yougoslavia,... please update.

Reply:
This list should have all the old iso cold? Also, the `old_iso` dictionary will be quite long, maybe worth to put them in the constant files?

Reply:
Yes, this list should be correct. As @tovogt argued below, it is a good idea to either put them in the constant files, propose a pull request to include old iso3 to the pacakge `iso3166` or use `pycountry` .